modepy/nodes.py

```diff
@@ -66,14 +67,18 @@
 
 # {{{ equidistant nodes
 
-def equidistant_nodes(dims, n, node_tuples=None):
+def equidistant_nodes(
+        dims: int, n: int,
+        node_tuples: Optional[Sequence[Tuple[int, ...]]] = None
+        ) -> np.ndarray:
     """
-    :arg dims: dimensionality of desired simplex
+    :param dims: dimensionality of desired simplex
```

Comment:
:cry: ?

Not because I have a particular preference... (Is there a reason to prefer one over the other? Any documented suggestions?) but mostly because 90% of everything else uses `:arg`, and I'm reluctant to start another consistency crusade. OTOH, this one is fairly regex-able...

Reply:
Hm, at some point I seem to remember that `arg` didn't pick up typing annotations but `param` did. It doesn't seem like that's the case anymore though (just checked).

Reply:
Ha! Apparently [sphinx-autodoc-typehints](https://github.com/agronholm/sphinx-autodoc-typehints#how-it-works) liked `param`. The version that made it into the main repo probably removed that restriction.

Reply:
Replaced in 5ef3e73.